README.md

```diff
@@ -173,3 +173,11 @@ During the build process, a list of warnings and errors is displayed that should
 The favicon of the company website is used as the company icon. It can be checked using our icon service:
 
 [https://icons.adguard.org/icon?domain=adguard.com](https://icons.adguard.org/icon?domain=adguard.com)
+
+## Policy
+
+| **Categories**                                                                                          | **Restrictions**                                         |
+|---------------------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| misc, audio_video_player, customer_interaction, social_media                                            | availability of a law firm, sufficient popularity of use |
+| pornvertising, advertising, essential, site_analytics, cdn, hosting, email, telemetry, mobile_analytics | will be added without restrictions                       |
+| stand-alone sites not related to any service                                                            | will not be added                                        |
```

Comment:
This kind of sounds "too much". What we're trying to solve here is that people sometime may bring a personal blog or a small website. Maybe we should simply write about that?

Something like: "We are yet to come up with detailed policy. For now, we can make decision on inclusion a new tracker based on the personal preference. For example, we avoid adding personal websites/blogs, or services that do not appear to be popular enough."

Reply:
@ameshkov Would it be easier to use a rule similar to that of the [AdGuard filter policy](https://adguard.com/kb/general/ad-filtering/filter-policy/#common-criteria) in terms of popularity? It seems to be working well so far.

> Website-specific rules will be added only if a website has sufficient traffic. Traffic is determined by open statistics (if available) or by other means, such as followers on social media. A website's traffic is considered sufficient when it has 100 thousand visits per month. We will consider adding a rule for a website that's not that popular, but the final decision is up to the filter list maintainer.

Reply:
Good idea, putting a query cap on won't be future proof, so an alternative would be to use something like Cloudflare's domain ranking api to get the top x amount.

https://developers.cloudflare.com/radar/investigate/domain-ranking-datasets/